docs/quickstart.md

```diff
@@ -176,7 +195,7 @@ module.exports = async function (context) {
 ```
 1. May core tools version may block debugging if no storage account is defined as some Azure Functions triggers depend on them.  Dapr triggers do not, but to get around this validation edit the `local.settings.json` file and add `"none"` as the value for `AzureWebJobsStorage`.  A fix for this workaround is [being tracked here](https://github.com/Azure/azure-functions-core-tools/issues/2065).
 1. Save and run the app.  Note that because we now need to recieve trigger events for the dapr sidecar, we need to define the port the function will listen on for these triggers.  
-    `dapr run --app-id function-app --app-port 3001 --port 3501 -- func start -p 7071`
+    `dapr run --app-id function-app --app-port 3001 --port 3501 -- func start -p 7071` or starting the function debugger and then starting dapr with `dapr run --app-id function-app --app-port 3001 --port 3501`.
```

Comment:
I think this is the problem that I ran into. I could not get both Dapr and the VS Code debugger to work together and it must be a combination of env variable and start order. It would be good to be clear on a consistent way

Reply:
Correct. Adding a line about this